docs/MO_DG/prepare_model/convert_model/mxnet_specific/Convert_Style_Transfer_From_MXNet.md

```diff
@@ -1,117 +1,117 @@
 # Convert MXNet Style Transfer Model {#openvino_docs_MO_DG_prepare_model_convert_model_mxnet_specific_Convert_Style_Transfer_From_MXNet}
 
-The tutorial explains how to generate a model for style transfer using the public MXNet\* neural style transfer sample.
-To use the style transfer sample from OpenVINO&trade;, follow the steps below as no public pre-trained style transfer model is provided with the OpenVINO toolkit.
+This article provides instructions on how to generate a model for style transfer using the public MXNet neural style transfer sample.
 
-#### 1. Download or clone the repository with an MXNet neural style transfer sample: [Zhaw's Neural Style Transfer repository](https://github.com/zhaw/neural_style).
+**Step 1**: Download or clone the repository [Zhaw's Neural Style Transfer repository](https://github.com/zhaw/neural_style) with an MXNet neural style transfer sample.
 
-#### 2. Prepare the environment required to work with the cloned repository:
-1. Install packages dependency:<br>
+**Step 2**: Prepare the environment required to work with the cloned repository:
+
+> **NOTE**: Python-tk installation is needed only for Linux. Python for Windows includes it by default.
+
+1. Install packages dependency.<br>
 ```sh
 sudo apt-get install python-tk
 ```
 
-Installing python-tk step is needed only for Linux, as it is included by default in Python\* for Windows\*.
-
-2. Install Python\* requirements:
+2. Install Python requirements:
 ```sh
 pip3 install --user mxnet
 pip3 install --user matplotlib
 pip3 install --user scikit-image
 ```
 
-#### 3. Download the pre-trained [VGG19 model](https://github.com/dmlc/web-data/raw/master/mxnet/neural-style/model/vgg19.params) and save it to the root directory of the cloned repository because the sample expects the model `vgg19.params` file to be in that directory.<br>
+**Step 3**: Download the pre-trained [VGG19 model](https://github.com/dmlc/web-data/raw/master/mxnet/neural-style/model/vgg19.params) and save it to the root directory of the cloned repository. The sample expects the model *`vgg19.params`* file to be in that directory.<br>
```

Comment:
```suggestion
**Step 3**: Download the pretrained [VGG19 model](https://github.com/dmlc/web-data/raw/master/mxnet/neural-style/model/vgg19.params) and save it to the root directory of the cloned repository. The sample expects the model *`vgg19.params`* file to be in that directory.<br>
```
I guess we all need to decide whether "pre-train" or "pretrain".  I'm an advocate of using the latter ;)

Reply:
I'm for "pre-train" but let's wait for rest to voice their preferences :)

Reply:
Exactly ;)